import interface with absolute path

I got the following error in my project (using TypeScript 4.6.3):
```
node_modules/codemirror-graphql/variables/hint.d.ts:3:24 - error TS2307: Cannot find module 'src/hint' or its corresponding type declarations.

3 import { IHints } from 'src/hint';
                         ~~~~~~~~~~
```

I traced it back to this `import` statement:

https://github.com/graphql/graphiql/blob/main/packages/codemirror-graphql/src/variables/hint.ts#L22

```
import { IHints } from 'src/hint';
```

My fix would be a relative import:

```
import { IHints } from '../hint';
```

--- a/.changeset/witty-owls-sit.md
+++ b/.changeset/witty-owls-sit.md
@@ -1,0 +1,5 @@
+---
+'codemirror-graphql': patch
+---
+
+Fix typescript path resolution bug in codemirror-graphql

--- a/packages/codemirror-graphql/src/utils/hintList.ts
+++ b/packages/codemirror-graphql/src/utils/hintList.ts
@@ -8,7 +8,7 @@
  */
 
 import type CodeMirror from 'codemirror';
-import { IHint, IHints } from 'src/hint';
+import { IHint, IHints } from '../hint';
 
 // Create the expected hint response given a possible list and a token
 export default function hintList(

--- a/packages/codemirror-graphql/src/utils/info-addon.ts
+++ b/packages/codemirror-graphql/src/utils/info-addon.ts
@@ -8,7 +8,7 @@
  */
 
 import CodeMirror from 'codemirror';
-import { GraphQLInfoOptions } from 'src/info';
+import { GraphQLInfoOptions } from '../info';
 
 CodeMirror.defineOption(
   'info',

--- a/packages/codemirror-graphql/src/utils/jump-addon.ts
+++ b/packages/codemirror-graphql/src/utils/jump-addon.ts
@@ -8,7 +8,7 @@
  */
 
 import CodeMirror from 'codemirror';
-import { GraphQLJumpOptions } from 'src/jump';
+import { GraphQLJumpOptions } from '../jump';
 
 CodeMirror.defineOption(
   'jump',

--- a/packages/codemirror-graphql/src/variables/hint.ts
+++ b/packages/codemirror-graphql/src/variables/hint.ts
@@ -19,7 +19,7 @@ import {
   GraphQLInputFieldMap,
 } from 'graphql';
 import type { State, Maybe } from 'graphql-language-service';
-import { IHints } from 'src/hint';
+import { IHints } from '../hint';
 
 import forEachState from '../utils/forEachState';
 import hintList from '../utils/hintList';

